requirements/default.txt

```diff
@@ -229,3 +229,9 @@ markus==2.1.0 \
 backoff==1.10.0 \
     --hash=sha256:5e73e2cbe780e1915a204799dba0a01896f45f4385e636bcca7a0614d879d0cd \
     --hash=sha256:b8fba021fac74055ac05eb7c7bfce4723aedde6cd0a504e5326bcb0bdd6d19a4
+structlog==20.1.0 \
+    --hash=sha256:7a48375db6274ed1d0ae6123c486472aa1d0890b08d314d2b016f3aa7f35990b \
+    --hash=sha256:8a672be150547a93d90a7d74229a29e765be05bd156a35cdcc527ebf68e9af92
+colorama==0.4.3 \
```

Comment:
`colorama` is a dev-only requirement, but not the first (`waitress`, etc). The color really helps.